### PR TITLE
cli support for routegroups

### PR DESCRIPTION
**Pull request by scott-cotton**
draft cli for routegroup support.

```shell
===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup list
NAME   ROUTING KEY   CLUSTER   CREATED   STATUS

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup apply -f ~/temp/cli/localhost/rg/rg.yaml --set name=rg-x1
Created routegroup "rg-x1" (routing key: 3rt3cxbk6k0hb) in cluster "local-cluster".

Waiting (up to --wait-timeout=3m0s) for route group to be ready...
✓ Route group status: Ready: all 2 sandboxes are ready

Dashboard page: https://app.signadot.com

ROUTEGROUP ENDPOINT     TARGET                            URL
frontend-svc-endpoint   http://frontend.hotrod.svc:8080   http://frontend-svc-endpoint--rg-x1.localhost.signadot.com:8086
route-svc-endpoint      http://route.hotrod.svc:8083      http://route-svc-endpoint--rg-x1.localhost.signadot.com:8086

The routegroup "rg-x1" was applied and is ready.
===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup list
NAME    ROUTING KEY     CLUSTER         CREATED                STATUS
rg-x1   3rt3cxbk6k0hb   local-cluster   2022-12-23T01:11:08Z   Ready

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup list -o yaml
- createdAt: "2022-12-23T01:11:08Z"
  endpoints:
  - name: frontend-svc-endpoint
    target: http://frontend.hotrod.svc:8080
    url: http://frontend-svc-endpoint--rg-x1.localhost.signadot.com:8086
  - name: route-svc-endpoint
    target: http://route.hotrod.svc:8083
    url: http://route-svc-endpoint--rg-x1.localhost.signadot.com:8086
  name: rg-x1
  routingKey: 3rt3cxbk6k0hb
  spec:
    cluster: local-cluster
    endpoints:
    - name: frontend-svc-endpoint
      target: http://frontend.hotrod.svc:8080
    - name: route-svc-endpoint
      target: http://route.hotrod.svc:8083
    match:
      any:
      - label:
          key: rg-feature
          value: rg-*
  status:
    matchedSandboxes:
    - sb-fe
    - sb-ro
    message: all 2 sandboxes are ready
    ready: true
    reason: AllSandboxesReady

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup get rg-x1 -o yaml
createdAt: "2022-12-23T01:11:08Z"
endpoints:
- name: frontend-svc-endpoint
  target: http://frontend.hotrod.svc:8080
  url: http://frontend-svc-endpoint--rg-x1.localhost.signadot.com:8086
- name: route-svc-endpoint
  target: http://route.hotrod.svc:8083
  url: http://route-svc-endpoint--rg-x1.localhost.signadot.com:8086
name: rg-x1
routingKey: 3rt3cxbk6k0hb
spec:
  cluster: local-cluster
  endpoints:
  - name: frontend-svc-endpoint
    target: http://frontend.hotrod.svc:8080
  - name: route-svc-endpoint
    target: http://route.hotrod.svc:8083
  match:
    any:
    - label:
        key: rg-feature
        value: rg-*
status:
  matchedSandboxes:
  - sb-fe
  - sb-ro
  message: all 2 sandboxes are ready
  ready: true
  reason: AllSandboxesReady

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup get rg-x1
Name:             rg-x1
Routing Key:      3rt3cxbk6k0hb
Cluster:          local-cluster
Created:          Thu, 22 Dec 2022 17:11:08 PST (About a minute ago)
Dashboard page:   https://app.signadot.com
Status:           Ready (AllSandboxesReady: all 2 sandboxes are ready)

ROUTEGROUP ENDPOINT     TARGET                            URL
frontend-svc-endpoint   http://frontend.hotrod.svc:8080   http://frontend-svc-endpoint--rg-x1.localhost.signadot.com:8086
route-svc-endpoint      http://route.hotrod.svc:8083      http://route-svc-endpoint--rg-x1.localhost.signadot.com:8086

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup delete rg-x1
Deleted routegroup "rg-x1".

Waiting (up to --wait-timeout=5m0s) for routegroup to finish terminating...
✓ RouteGroup status: Terminated

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup get rg-x1
Error: 400 Bad Request: unable to fetch routegroup: not found

===================================================================

./signadot --config ~/.signadot/localhost.yaml routegroup list
NAME   ROUTING KEY   CLUSTER   CREATED   STATUS
```

### CODE REVIEW

File: go.mod
Comment (jmsktm) [inline go.mod:44]: Need to release go-sdk and update version here before merge.

**Review by foxish — APPROVED** [review]

**Comment by foxish** [comment]
LGTM after the change to pin the version of the go-sdk

**Pull request merged**
